sdk/python/jobs/finetuning/standalone/model-as-a-platform/chat/chat_completion_with_model_as_platform.ipynb

```diff
@@ -0,0 +1,565 @@
+   },
+   "outputs": [],
+   "source": [
+    "from azure.ai.ml import MLClient\n",
+    "from azure.identity import (\n",
+    "    DefaultAzureCredential,\n",
+    "    InteractiveBrowserCredential,\n",
+    ")\n",
+    "from azure.ai.ml.constants._common import AssetTypes"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Create AzureML Workspace connections"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "from azure.ai.ml import MLClient\n",
+    "from azure.identity import (\n",
+    "    DefaultAzureCredential,\n",
+    "    InteractiveBrowserCredential,\n",
+    ")\n",
+    "\n",
+    "try:\n",
+    "    credential = DefaultAzureCredential()\n",
+    "    credential.get_token(\"https://management.azure.com/.default\")\n",
+    "except Exception as ex:\n",
+    "    credential = InteractiveBrowserCredential()\n",
+    "\n",
+    "try:\n",
+    "    workspace_ml_client = MLClient.from_config(credential=credential)\n",
+    "except:\n",
+    "    workspace_ml_client = MLClient(\n",
+    "        credential,\n",
+    "        subscription_id=\"<SUBSCRIPTION_ID>\",\n",
+    "        resource_group_name=\"<RESOURCE_GROUP>\",\n",
+    "        workspace_name=\"<WORKSPACE_NAME>\",\n",
+    "    )\n",
+    "\n",
+    "# the models, fine tuning pipelines and environments are available in various AzureML system registries,\n",
+    "# Example: Phi family of models are in \"azureml\"\n",
+    "registry_ml_client = MLClient(credential, registry_name=\"azureml\")\n",
+    "\n",
+    "# Get AzureML workspace object.\n",
+    "workspace = workspace_ml_client._workspaces.get(workspace_ml_client.workspace_name)\n",
+    "workspace.id"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 2. Pick a foundation model to fine tune\n",
+    "\n",
+    "`Phi-3-mini-4k-instruct` is a 3.8B parameters, lightweight, state-of-the-art open model built upon datasets used for Phi-2. The model belongs to the Phi-3 model family, and the Mini version comes in two variants 4K and 128K which is the context length (in tokens) it can support. You can browse these models in the Model Catalog in the Azure AI Studio, filtering by the `chat-completion` task. In this example, we use the `Phi-3-mini-4k-instruct` model. If you have opened this notebook for a different model, replace the model name and version accordingly.\n",
+    "\n",
+    "Note the model id property of the model. This will be passed as input to the fine tuning job. This is also available as the `Asset ID` field in model details page in Azure AI Studio Model Catalog."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model_name = \"Phi-3-mini-4k-instruct\"\n",
+    "foundation_model = registry_ml_client.models.get(model_name, label=\"latest\")\n",
```

Comment:
how about just 'model' instead of 'foundation_model' ?

Reply:
Changed.